app/experimenter/static/rapid/components/forms/ExperimentForm.tsx

```diff
@@ -219,6 +219,28 @@ export const BranchesForm: React.FC = () => {
     dispatch(updateExperiment({ variants: updatedVariants }));
   };
 
+  const NEW_BRANCH = {
+    name: "",
```

Comment:
I would be nice to give a default name to each branch of `variant-${i}` (assuming it's not already taken)